es6-promises/index.js

```diff
@@ -0,0 +1,8 @@
+const takeAChance = require('./take-a-chance');
+const chances = takeAChance('moamel');
+chances.then(value => {
+  console.log(value);
+});
+chances.catch(error => {
+  console.error(error);
+});
```

Comment:
Please update your `catch` to only print out the error message, not the whole stack dump.